frontend/src/app/services/settings/t4c-instance.service.ts

```diff
@@ -58,6 +61,12 @@ export class T4CInstanceService {
   );
   public readonly t4cInstance$ = this._t4cInstance.asObservable();
 
+  public readonly unarchivedT4cInstances$ = this._t4cInstances.pipe(
+    map((t4cInstances) =>
+      t4cInstances?.filter((t4cInstance) => !t4cInstance.is_archived)
+    )
+  );
```

Comment:
why is that needed? it seems to be unused to me?

Reply:
Good catch, deleted it.